ibis/backends/tests/test_window.py

```diff
@@ -1215,7 +1215,7 @@ def test_first_last(backend):
     raises=com.OperationNotDefinedError,
     reason="not support by the backend",
 )
-@pytest.mark.notyet(["flink"], raises=Py4JJavaError, reason="not supported by Flink")
+@pytest.mark.broken(["flink"], raises=Py4JJavaError, reason="bug in Flink")
```

Comment:
There were two issues here:

1. `time` is a reserved keyword
2. Even then, there seems to be an issue: https://apache-flink.slack.com/archives/C03G7LJTS2G/p1700675938430139

The stack trace on the second says, `Caused by: org.apache.flink.util.FlinkRuntimeException: org.apache.flink.api.common.InvalidProgramException: Table program cannot be compiled. This is a bug. Please file an issue.`, but I'm still discussing with somebody on the Flink side; may update later with a Jira.